### ISSUE TITLE
Consider using Elekto to run TOC elections

### ISSUE THREAD

**Issue by chipchilders** [cloudfoundry/community#135]
This seems smoother than using CIVS for election administration: https://github.com/elekto-io/elekto/

Two key benefits:
1) GitHub ID-based authentication, instead of having to source the "best" email address for each voter while not making the list of email addresses public
2) Alignment with k8s community's approach to elections (they sponsored the creation of Elekto, and use it)

Downside:
1) Having to host / run / manage the software vs just using CIVS

**Comment by emalm** [comment]
Would be cool to see whether we can run the app on CF! :D

**Comment by chipchilders** [comment]
Indeed... self hosting an app would at least allow us to resolve issues
like this. :)

**Comment by LeePorte** [comment]
@emalm has been busy https://github.com/cloudfoundry/community/pull/259

**Comment by emalm** [comment]
@LeePorte I can confirm that it was cool to [get to run on CF](https://github.com/cloudfoundry/community/pull/259#discussion_r862272355). 😎

**Comment by LeePorte** [comment]
We are now using Elekto
